transbank/webpay/webpayplus/transaction.py

```diff
@@ -56,3 +58,37 @@ def create(cls, buy_order, session_id, amount,
             else:
                 raise TransactionCreateException(http_response.status_code, e.args)
 
+    @classmethod
+    def commit(cls, token_ws, options=None):
+        commerce_code = None
```

Comment:
Seems like all this logic you use to prepare the Options object and headers are same... Do not could it be in reusable method?